### PR TITLE
Add StatsD metrics 

### PR DESCRIPTION
**Pull request by chris-wood**
Builds on #24.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -178,6 +179,26 @@ func main() {
 		gateway: gateway,
 	}
 
+	// Configure metrics
+	metricsHost := os.Getenv(statsdHostVariable)
+	metricsPort := os.Getenv(statsdPortVariable)
+	metricsTimeout, err := strconv.ParseInt(os.Getenv(statsdTimeoutVariable), 10, 64)
+	if err != nil {
+		log.Fatalf("Failed parsing metrics timeout: %s", err)
+		metricsTimeout = 100
```
Comment (Mixser) [inline main.go:188]: This line could not be reached, because `Fatalf` will call `os.Exit(1)`

File: gateway.go [hunk 2]
```diff
@@ -52,11 +61,19 @@ func (s *gatewayResource) gatewayHandler(w http.ResponseWriter, r *http.Request)
 	if s.verbose {
 		log.Printf("%s Handling %s\n", r.Method, r.URL.Path)
 	}
+
+	metrics := s.metricsFactory.Create(metricsEventGatewayRequest)
+	metrics = s.metricsFactory.Create(metricsEventGatewayRequest)
```
Comment (Mixser) [inline gateway.go:66]: it's looks like a rudimental (three equal lines)

Comment (chris-wood) [reply gateway.go]: Oops, meant to remove those. Fixed!

File: main.go (hunk 1 above)
Comment (chris-wood) [reply main.go:188]: Good catch! Fixed.

**Review by Mixser — APPROVED** [review]

**Pull request merged**